### DISCUSSION TITLE
Hitting "Play Next" Endlessly

### DISCUSSION THREAD

**Discussion by hepheastus** [stashapp/stash#3833] [Q&A] [answered]
I've been trying to figure out how to hit next endlessly.
Right now, I can only hit next 40 times before I have to either hit shuffle in the queue log or go back to scenes (which I have defaulted to random) and click the first scene.  I have even tried to hit the bottom down arrow button in the queue list or even making my default scene show 1000 files instead of 40 and it still defaults to 40.

I just wanna keep hitting next forever.

**Comment by WithoutPants** [comment, marked as answer]
I've raised a bug to track this: https://github.com/stashapp/stash/issues/3834

**Comment by hepheastus** [comment]
Thanks!
